paperqa/docs.py

```diff
@@ -294,12 +293,15 @@ async def aadd(  # noqa: PLR0912
             )
             if not texts:
                 raise ValueError(f"Could not read document {path}. Is it empty?")
-            result = await llm_model.run_prompt(
-                prompt=parse_config.citation_prompt,
-                data={"text": texts[0].text},
-                system_prompt=None,  # skip system because it's too hesitant to answer
+            messages = [
+                Message(
+                    content=parse_config.citation_prompt.format(text=texts[0].text)
+                ),
+            ]
+            result = await llm_model.call_single(
+                messages=messages,
```

Comment:
```suggestion
            result = await llm_model.call_single(
                messages=[
                    Message(
                        content=parse_config.citation_prompt.format(text=texts[0].text)
                    ),
                ],
```

If you're going to use keyword argument in `call_single`, imo no need to have the `messages` local too